internal/operands/tekton-pipelines/reconcile_test.go

```diff
@@ -202,6 +202,53 @@ var _ = Describe("environments", func() {
 		})
 	})
 
+	Context("With openshift-cnv namespace in ssp CR for pipelines", func() {
+		BeforeEach(func() {
+			request.Instance.Spec.FeatureGates.DeployTektonTaskResources = true
+			request.Instance.Spec.TektonPipelines = &ssp.TektonPipelines{
+				Namespace: openshiftCNVNamespace,
+			}
+		})
+
+		It("kubevirt.io/deploy-namespace annotation in configMaps should not be replaced", func() {
+			_, err := operand.Reconcile(request)
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, configMap := range bundle.ConfigMaps {
+				objNamespace := namespace
+
+				if configMap.Name == "test-cm" {
+					configMap.Namespace = testDifferentNamespace
+					objNamespace = testDifferentNamespace
+				}
+
+				key := client.ObjectKeyFromObject(&configMap)
+				cm := &v1.ConfigMap{}
+				ExpectWithOffset(1, request.Client.Get(request.Context, key, cm)).ToNot(HaveOccurred())
```

Comment:
Using `ExpectWithOffset()` here does not make sense, because this is the top level of the test case.